datafusion/sqllogictest/test_files/explain_tree.slt

```diff
@@ -519,6 +519,52 @@ physical_plan
 17)│       format: arrow       │
 18)└───────────────────────────┘
 
+# Query for sort.
+query TT
+explain SELECT * FROM table1 ORDER BY string_col;
+----
+logical_plan
+01)Sort: table1.string_col ASC NULLS LAST
+02)--TableScan: table1 projection=[int_col, string_col, bigint_col, date_col]
+physical_plan
+01)┌───────────────────────────┐
+02)│          SortExec         │
+03)│    --------------------   │
+04)│         sort keys:        │
+05)│  [string_col@1 ASC NULLS  │
+06)│            LAST]          │
+07)└─────────────┬─────────────┘
+08)┌─────────────┴─────────────┐
+09)│       DataSourceExec      │
+10)│    --------------------   │
+11)│          files: 1         │
+12)│        format: csv        │
+13)└───────────────────────────┘
+
+# Query for sort with limit.
+query TT
+explain SELECT * FROM table1 ORDER BY string_col LIMIT 1;
+----
+logical_plan
+01)Sort: table1.string_col ASC NULLS LAST, fetch=1
+02)--TableScan: table1 projection=[int_col, string_col, bigint_col, date_col]
+physical_plan
+01)┌───────────────────────────┐
+02)│          SortExec         │
+03)│    --------------------   │
+04)│          limit: 1         │
+05)│                           │
+06)│         sort keys:        │
+07)│  [string_col@1 ASC NULLS  │
+08)│            LAST]          │
```

Comment:
I wonder if there is some way to avoid the redundancy of `sort keys` 🤔 

For example I think this would be look better (not sure how to do it)

```
01)┌───────────────────────────┐
02)│          SortExec         │
03)│    --------------------   │
07)│   string_col@1 ASC NULLS  │
08)│            LAST           │
05)│                           │
04)│          limit: 1         │
09)└─────────────┬─────────────┘